.github/workflows/to.api.contract.tests.yml

```diff
@@ -64,6 +58,12 @@ jobs:
         - 5432:5432
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
 
+    env:
+      # alpine:3.13
+      ALPINE_VERSION: sha256:08d6ca16c60fe7490c03d10dc339d9fd8ea67c6466dea8d558526b1330a85930
+      GOPATH: ${{ github.workspace }}/go
+      PATH: ${{ env.GOPATH }}/bin:${{ env.PATH }}
```

Comment:
This won't work, because `GOPATH` is not set when it is referenced in `${{ env.GOPATH }}`. You can modify `PATH` in the environment in a step like this:
https://github.com/apache/trafficcontrol/blob/a1bd413864feef5a7b87228cc80f087cf1680812/.github/workflows/container-trafficserver-alpine.yml#L66

That said, you probably do not need to set `PATH` at all, right?

Reply:
Oh I see, you did take the `PATH`-setting part out.